### PR TITLE
Test build of Eve7 components - DO NOT MERGE!!!

### PR DESCRIPTION
**Pull request by linev**

### CODE REVIEW

File: etc/eve7/Colors.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+			turquoise: '40e0d0',
+			violet: 'ee82ee',
+			wheat: 'f5deb3',
+			white: 'ffffff',
+			whitesmoke: 'f5f5f5',
+			yellow: 'ffff00',
+			yellowgreen: '9acd32',
+			transparent: '00000000'
+   };// colorButton colors
+
```
Comment (bellenot) [inline etc/eve7/Colors.js]: Indentation ? Could you try to not use tabs, but spaces?

File: etc/eve7/Component-preload.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+			}
+		},
+
+		onAfterRendering : function() {
+		}
+	});
+
+	return Component;
+
+});
```
Comment (bellenot) [inline etc/eve7/Component-preload.js]: Here again: Indentation

File: etc/eve7/Component.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+			}
+		},
+
+		onAfterRendering : function() {
+		}
+	});
+
+	return Component;
+
+});
```
Comment (bellenot) [inline etc/eve7/Component.js]: Same here (indentation)

File: graf3d/eve7/glu/alg-outline [hunk 4]
```diff
@@ -0,0 +1,228 @@
+/*
```
Comment (bellenot) [inline graf3d/eve7/glu/alg-outline]: Do you really need to add all those glu files?

**Comment by bellenot** [comment]
I'll go through the C++ source later...

**Comment by linev** [comment]
> I'll go through the C++ source later...

@bellenot and all other
It is really build test - I used very fancy forwarding of template declaration with 6 arguments.
Want to be sure that it works on all platforms.
On the meeting we should discuss how deep we want to refactor old code

**Comment by bellenot** [comment]
@phsft-bot build with flags -Droot7=ON

**Comment by linev** [comment]
@phsft-bot build with flags -Droot7=ON -Dcxx14=ON

**Comment by linev** [comment]
Real PR will follow soon

**Pull request closed without merging**